vaadinfrontend/src/main/java/life/qbic/datamanager/views/projects/project/experiments/ExperimentInformationPage.java

```diff
@@ -47,72 +42,71 @@
-      @Autowired ExperimentDetailsComponent experimentDetailsComponent,
+      @Autowired ExperimentSupportComponent experimentSupportComponent,
+      @Autowired ExperimentMainComponent experimentMainComponent,
       @Autowired ProjectInformationService projectInformationService,
       @Autowired ExperimentInformationService experimentInformationService) {
     Objects.requireNonNull(projectNavigationBarComponent);
-    Objects.requireNonNull(experimentDetailsComponent);
+    Objects.requireNonNull(experimentSupportComponent);
+    Objects.requireNonNull(experimentMainComponent);
     Objects.requireNonNull(projectInformationService);
-    this.experimentInformationService = Objects.requireNonNull(experimentInformationService);
-    setupBoard(projectNavigationBarComponent, experimentDetailsComponent);
+    Objects.requireNonNull(experimentInformationService);
+    setupBoard(projectNavigationBarComponent, experimentMainComponent, experimentSupportComponent);
     experimentInformationPageHandler = new ExperimentInformationPageHandler(
-        projectNavigationBarComponent, experimentDetailsComponent,
-        projectInformationService);
+        projectNavigationBarComponent, experimentMainComponent, experimentSupportComponent,
+        projectInformationService, experimentInformationService);
     log.debug(String.format(
-        "\"New instance for Experiment Information page (#%s) created with Project Navigation Bar Component (#%s) and Experiment Details Component (#%s)",
+        "\"New instance for ExperimentInformationPage (#%s) created with ProjectNavigationBar Component (#%s), ExperimentMain component (#%s) and ExperimentSupport component (#%s)",
         System.identityHashCode(this), System.identityHashCode(projectNavigationBarComponent),
-        System.identityHashCode(experimentDetailsComponent)
-      ));
+        System.identityHashCode(experimentMainComponent),
+        System.identityHashCode(experimentSupportComponent))
+    );
   }
 
   private void setupBoard(ProjectNavigationBarComponent projectNavigationBarComponent,
-      ExperimentDetailsComponent experimentDetailsComponent) {
-    Board board = new Board();
-
-    Row rootRow = new Row();
-
-    VerticalLayout mainComponents = new VerticalLayout();
-    mainComponents.setPadding(false);
-    mainComponents.setMargin(false);
-    mainComponents.setSpacing(false);
-    mainComponents.add(projectNavigationBarComponent, experimentDetailsComponent);
-
-    rootRow.add(mainComponents, 3);
-    itemCollection = ExperimentItemCollection.create("Add a new experiment");
-    rootRow.add(itemCollection, 1);
-
-    board.add(rootRow);
-
-    board.setSizeFull();
-
-    add(board);
+      ExperimentMainComponent experimentMainComponent,
+      ExperimentSupportComponent experimentSupportComponent) {
+    this.addClassName("experiment-page");
+    this.add(projectNavigationBarComponent);
+    this.add(experimentMainComponent);
+    this.add(experimentSupportComponent);
```

Comment:
you can omit the `this` reference, since the compiler knows how to call the inherited methods here anyway.

Reply:
Good point